BoxContentSDK/BoxContentSDK/Requests/BOXTrashedFolderRestoreRequest.h

```diff
@@ -7,6 +7,18 @@
 
 @interface BOXTrashedFolderRestoreRequest : BOXRequest
 
+// NOTE: Both the associateID and requestDirectoryPath values are required for performing the request in the background.
```

Comment:
Maybe add `This is required…` note to `requestDirectoryPath`

Reply:
done